### DISCUSSION TITLE
Small gap between perimeter, but only in the first few mm?

### DISCUSSION THREAD

**Discussion by gitfve** [mriscoc/Ender3V2S1#1157] [General]
Small gap between perimeter, but only in the first few mm?

I used a lot debugging time on this, and now i need some hints.

-Ender3 S1 PRO, Direct extruder, printed with PLA, using Jerk (NOT Linear Advance).

All my print is nice i think most wont see my problem as a problem.

-When printing perimeter, i see the first few mm is a little thinner then the rest of the perimeter. It make small play or gap between them, but only in the first few mm.
-All printed with NO retraction or other pull back of filament, and no move over surface.
-You see it when changing direction as printing a cube (direction change 90 degree)

I have tested so much, one thing a time. 
Temperature, Layer-height, Extrusion-width, Speed, Jerk, Acceleration, and more. Nothing make any difference, it is exactly the same.

-Gcode, i have inspected the Gcode and it is not the problem, perimeter is with small overlap and placed right.

The extruder is calibrated and all print nice, just not the first few mm in the perimeter. 
You only see it in a magnifier, but a screwdriver can separate the walls.

For med it look like the extruder is a little behind the move in x,y direction (under extrusion in the first few mm).

Using firmware: (Marlin based) mriscoc Professional Firmware 20230904.

I attach a picture, focus is not so good, but look in the yellow highlights, the part is small the width=5mm.
Pic to the left, you see 3 perimeter and a little thinner in the start.
Pic to the right, i put a screwdriver between the walls. 
![Perimeter-Gap](https://github.com/mriscoc/Ender3V2S1/assets/6228825/28606f3a-b49e-4665-99b7-79cb7679080b)

I only see two things there can do it: 
The printer itself, or the the firmware?
(It is not the firmware. I installed "Ender-3S1_Pro_HWv24S1_301_SWV2.0.8.26F4_F401_FDM_LASER.bin", from Creality. The same Gcode file did not print any better. Glad to get back to mriscoc fw again.)

I have sliced the part in Prusa 2.6.0 and 2.7.0 and S3D with same result. 

If anyone have a Ender3_S1_PRO i can upload the small *.Gcode file it only that 4 minutes to make the test.

What can that be?

**Comment by tmuka** [comment]
I'm no expert, but my impression was printing a "skirt" is meant to help prime the extruder filament pressure ahead of your part.  Have you tried printing a skirt to see if that makes a difference?
